### DISCUSSION TITLE
can't access most websites while using mitmproxy

### DISCUSSION THREAD

**Discussion by kshitij86** [mitmproxy/mitmproxy#5471] [General]
after setting up `mitmproxy`, I'm not able to access many webapps that are needed to be used on a regular basis. I think it is Firefox issue as it detects `mitmproxy` and does not allow me to continue to the webpage even in the advanced settings, it only gives me a go back option. What can I do to resolve this?

![image](https://user-images.githubusercontent.com/26821140/180204856-760492db-6d96-4225-a4de-7fb6db0b7985.png)

![image](https://user-images.githubusercontent.com/26821140/180204791-f6c137d3-cb88-4c24-af21-27af9051a24a.png)

**Comment by kshitij86** [comment]
@mhils

**Reply by mhils** [reply]
Please do not ping me directly, that's totally unnecessary.

**Comment by mhils** [comment]
You need to install the mitmproxy CA in Firefox (which has its own certificate store). You can follow the instructions on <http://mitm.it/> for that.
